commands/command_fetch.go

```diff
@@ -107,11 +107,27 @@ func fetchCommand(cmd *cobra.Command, args []string) {
 }
 
 func pointersToFetchForRef(gitscanner *lfs.GitScanner, ref string) ([]*lfs.WrappedPointer, error) {
-	pointerCh, err := gitscanner.ScanTree(ref)
-	if err != nil {
+	var pointers []*lfs.WrappedPointer
+	var multiErr error
+	tempgitscanner := lfs.NewGitScanner(func(p *lfs.WrappedPointer, err error) {
+		if err != nil {
+			if multiErr != nil {
+				multiErr = fmt.Errorf("%v\n%v", multiErr, err)
```

Comment:
Coming back to this, if `CombineErrors` were to go in the `errors` package, I think it'd be better to just call it `errors.Combine`/